src/test/java/com/gojek/beast/converter/ConsumerRecordConverterIntegrationTest.java

```diff
@@ -87,6 +100,64 @@ public void shouldGetRecordForBQFromConsumerRecords() throws InvalidProtocolBuff
         assertEquals(record2ExpectedColumns, record2Columns);
     }
 
+    @Test
+    public void shouldIgnoreNullRecords() throws InvalidProtocolBufferException {
+        OffsetInfo record1Offset = new OffsetInfo("topic1", 1, 101, Instant.now().toEpochMilli());
+        OffsetInfo record2Offset = new OffsetInfo("topic1", 2, 102, Instant.now().toEpochMilli());
+        ConsumerRecord<byte[], byte[]> record1 = util.withOffsetInfo(record1Offset).createConsumerRecord("order-1", "order-url-1", "order-details-1");
+        ConsumerRecord<byte[], byte[]> record2 = util.withOffsetInfo(record2Offset).createEmptyValueConsumerRecord("order-2", "order-url-2");
+
+
+        Map<Object, Object> record1ExpectedColumns = new HashMap<>();
+        record1ExpectedColumns.put("bq_order_number", "order-1");
+        record1ExpectedColumns.put("bq_order_url", "order-url-1");
+        record1ExpectedColumns.put("bq_order_details", "order-details-1");
+        record1ExpectedColumns.putAll(util.metadataColumns(record1Offset, nowEpochMillis));
+
+
+        Map<Object, Object> record2ExpectedColumns = Collections.emptyMap();
+        List<ConsumerRecord<byte[], byte[]>> messages = Arrays.asList(record1, record2);
+
+        List<Record> records = recordConverter.convert(messages);
+
+        assertEquals(messages.size(), records.size());
+        Map<String, Object> record1Columns = records.get(0).getColumns();
+        Map<String, Object> record2Columns = records.get(1).getColumns();
+        assertEquals(record1ExpectedColumns.size(), record1Columns.size());
+        assertEquals(record1ExpectedColumns.size(), record1Columns.size());
```

Comment:
Did you mean `assertEquals(record2ExpectedColumns.size(), record2Columns.size());`

Reply:
Yeah, I copy-pasted it from existing line number 98. Fixing both.